python/paddle/v2/master/client.py

```diff
@@ -26,14 +26,27 @@ def set_dataset(self, paths):
             holder[idx] = c_ptr
         lib.paddle_set_dataset(self.c, holder, len(paths))
 
+    # return format: (record, errno)
+    # errno =  0: ok
+    #       = -1: EOF
```

Comment:
I don' think we need to return EOF. The user is using `next_record `, from the user's perspective, it's an infinite stream of data. EOF does not make sense here.
Actually, I am not convinced that we should return any error here. Current implementation is `next_record` will log error and try to recover from the error.
If we return error here, what can user do with it? (the retry logic is already inside `next_record`).